accounts-db/src/tiered_storage/footer.rs

```diff
@@ -133,7 +117,7 @@ pub struct TieredStorageFooter {
     /// The size of the footer including the magic number.
     pub footer_size: u64,
     // This field is persisted in the storage but not in this struct.
-    // The number should match FOOTER_MAGIC_NUMBER.
+    // The number should match FILE_MAGIC_NUMBER.
     // pub magic_number: u64,
```

Comment:
Magic number is not part of the footer as it is commented out.  The comment is to point out that there is one additional field after the footer (and the `magic_number: u64` is also commented out).  Maybe we can find a better place for this comment.

Reply:
> It's not part of the struct, but it is part of FOOTER_TAIL_SIZE, so we sometimes consider the magic number as part of the footer.

Aghh, got it. The concept of FOOTER_TAIL_SIZE is that the format of the last `FOOTER_TAIL_SIZE` bytes in any tiered-storage files will not change so that even if we have new versions of the footer that introduces new fields, we can still safely check the magic number and the format version.  The tail also has a field describing the size of the footer.